sdk/keyvault/.azure-pipelines/pr.client.yml

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT THIS FILE
+# This file is/may be generated automatically and any changes may be lost.
```

Comment:
Did you actually generate these or did you copy/paste them? At any case I would remove the "may be" part.

Reply:
I hand wrote them but included the warning to dissuade modifying them :P I'll make it more explicity even though they aren't automatically generated ... yet.